src/geo/torque-layer-view-base.js

```diff
@@ -0,0 +1,160 @@
+var _ = require('underscore');
+
+/**
+ * Implementation of all common logic of a torque-layer view.
+ * All the methods and attributes here need to be consistent across all implementing models.
+ *
+ * Methods are prefixed with _ to indicate that they are not intended to be used outside the implementing models.
+ */
+module.exports = {
+
+  _initialAttrs: function (layerModel) {
+    var extra = layerModel.get('extra_params');
+    return {
+      table: layerModel.get('table_name'),
+      user: layerModel.get('user_name'),
+      column: layerModel.get('property'),
+      blendmode: layerModel.get('torque-blend-mode'),
+      resolution: 1,
+      // TODO: manage time columns
+      countby: 'count(cartodb_id)',
+      sql_api_domain: layerModel.get('sql_api_domain'),
+      sql_api_protocol: layerModel.get('sql_api_protocol'),
+      sql_api_port: layerModel.get('sql_api_port'),
+      tiler_protocol: layerModel.get('tiler_protocol'),
+      tiler_domain: layerModel.get('tiler_domain'),
+      tiler_port: layerModel.get('tiler_port'),
+      maps_api_template: layerModel.get('maps_api_template'),
+      stat_tag: layerModel.get('stat_tag'),
+      animationDuration: layerModel.get('torque-duration'),
+      steps: layerModel.get('torque-steps'),
+      sql: this._getQuery(layerModel),
+      visible: layerModel.get('visible'),
+      extra_params: {
+        api_key: extra ? extra.map_key : ''
+      },
+      attribution: layerModel.get('attribution'),
+      cartocss: layerModel.get('cartocss') || layerModel.get('tile_style'),
```

Comment:
@javisantana considering the mention above, about some props that could be removed due to dropping backward compability, can the `tile_style` fallback be :fire: too?